### PR TITLE
Portuguese translations

### PR DESCRIPTION
**Pull request by matheusfelipeog**
I did the Music Player application translation 👍

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -7,13 +7,15 @@
     "en_EN": "Music Player",
     "nb_NO": "Musikkavspiller",
     "fr_FR": "Lecteur de musique",
-    "de_DE": "Musikplayer"
+    "de_DE": "Musikplayer",
+    "pt_BR": "Music Player"
```
Comment (andersevenrud) [inline metadata.json]: Is this not translatable either ?

Comment (matheusfelipeog) [reply metadata.json]: That word yes, but we use this "Music Player" frequently in Brazil. Do you want me to change?

Comment (andersevenrud) [reply metadata.json]: I see. I'll leave it up to you to decide then as this differs based on what region one lives in :)

Comment (matheusfelipeog) [reply metadata.json]: Then you can merge :)

**Pull request merged**